### DISCUSSION TITLE
Error in validator_handler.go

### DISCUSSION THREAD

**Discussion by mertensblackfyre** [earthboundkid/requests#73] [Support] [answered]
Is there is an error in  validator_handler.go.

`# github.com/carlmjohnson/requests
../../go/pkg/mod/github.com/carlmjohnson/requests@v0.23.3/validator_handler.go:23:17: undefined: errors.Join
note: module requires Go 1.20`

Please fix the bug. thank you

**Comment by earthboundkid** [comment, marked as answer]
You are using an old version of Go. Upgrade to Go 1.20.
